README.md

```diff
@@ -21,3 +26,83 @@ cloudformation yaml files with [pre-commit](https://pre-commit.com).
 Please install pre-commit. Once installed, the file validations will
 automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
+
+## Working with the Cluster
+
+### Reaching the Jumpbox
+In the AWS Console, find the jumpbox instance, and ssh it to using the PEM
+that was specified in the stack creation. For example,
+`ssh -i "toil.pem" ubuntu@10.23.95.18`.
+
+### Reaching the Leader
+Once you're on the Jumpbox, use this command to reach the cluster's leader,
+where zone is the zone where you created the jumpbox and leader and you
+specify the cluster name, for example, "rna-seq-reprocessing-toil-cluster-v001":
+`sudo toil ssh-cluster --zone us-east-1c rna-seq-reprocessing-toil-cluster-v001`
+
+### Setting up the Leader
+The leader requires a little setup. Our typical workflow is to update, install
+a few utilities, checkout the source code, and place the synapse configuration file.
+
+```
+# Update and Install
+apt-get update
+apt-get install -y git vim
+
+# Check out the project and switch to the relevant branch and location,
+# for example:
+
+mkdir ~/jobs && cd $_
+git clone https://github.com/Sage-Bionetworks/amp-workflows.git
+git checkout wpoehlmdev
+cd amp-workflows/amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/
+
+# Do any other necessary setup -- for example, add a .synapseConfig file
+mkdir /etc/synapse
+vi /etc/synapse/.synapseConfig
+
+# Finally, run the job! In the example from the repository mentioned above,
+# we have a job-runner script:
+nohup ./run-toil.py jobs/test-main-paired &
+```
+
+### Running a Toil Job on the Cluster
+See the Toil documentation on running workflows generally and
+[CWL](https://toil.readthedocs.io/en/latest/running/cwl.html?highlight=toil%20cwl%20runner)
+in particular; also, see the Toil documentation on
+[running a workflow in AWS](https://toil.readthedocs.io/en/latest/gettingStarted/quickStart.html#awscwl).
+
+The repository in the example above uses a [custom script](https://raw.githubusercontent.com/Sage-Bionetworks/amp-workflows/4e94f1fc1e3035e97b1b633565d71d496a072a20/amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/run-toil.py) to start a job.
```

Comment:
this is pointing to specific version of `run-toil.py`?  shouldn't it just point to master?

Reply:
Because they're not in master, unfortunately. I will have to remember to update these when they finally get merged. Waiting on others.